src/state/index.ts

```diff
@@ -2,6 +2,9 @@ import { connectRouter, RouterState } from 'connected-react-router';
 import { createBrowserHistory, History } from 'history';
 import { combineReducers, createStore, Reducer } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
+import { persistReducer, persistStore } from 'redux-persist';
+import autoMergeLevel1 from 'redux-persist/es/stateReconciler/autoMergeLevel1';
```

Comment:
Q: Can this be changed to `import { autoMergeLevel1 } from 'redux-persist/es/stateReconciler;`?

Reply:
They don't export these members. The type declaration file is wonky at best. Feel free to take a stab at it.

Reply:
They use default exports. I have imported them like so: 

```ts
import { default as autoMergeLevel1 } from 'redux-persist/es/stateReconciler/autoMergeLevel1';
```